android/src/main/java/com/swmansion/reanimated/ReanimatedModule.java

```diff
@@ -102,6 +103,21 @@ public NodesManager getNodesManager() {
     return mNodesManager;
   }
 
+  @ReactMethod(isBlockingSynchronousMethod = true)
+  public void installTurboModule() {
+    // When debugging in chrome the JS context is not available.
+    // https://github.com/facebook/react-native/blob/v0.67.0-rc.6/ReactAndroid/src/main/java/com/facebook/react/modules/blob/BlobCollector.java#L25
+    Utils.isChromeDebugger = getReactApplicationContext().getJavaScriptContextHolder().get() == 0;
+
+    if (!Utils.isChromeDebugger) {
+      this.getNodesManager().initWithContext(getReactApplicationContext());
+    } else {
+      Log.w(
```

Comment:
Seems like they use Chrome Debugger in this path, what's the point of showing a warning saying it should be ignored

Reply:
`Utils.isChromeDebugger` variable checks if context isn't available. If this is the only situation when it isn't available then we can delete the warning.

\cc @piaskowyk